tests/test_instance.py

```diff
@@ -121,6 +122,12 @@ async def test_profile_configuration_changes(instance):
     instance.profile = deepcopy(original_profile)
     assert not any(instance.configuration_changes())
 
+    # Regression test for bind mounts
+    instance.profile.home_mount = f"{Path.home()}/aiidalab_test"
+    instance.recreate()
+    assert await instance.status() is instance.AiidaLabInstanceStatus.CREATED
```

Comment:
I wonder whether this `assert await` is not also needed at the top of this function after `instance.create()`?